ui-participant/src/hub/HubPage.tsx

```diff
@@ -1,24 +1,43 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import { usePortalEnv } from '../providers/PortalProvider'
 import { useUser } from '../providers/UserProvider'
 
-import { Enrollee, ParticipantTask, Portal, Study } from '../api/api'
+import Api, { Enrollee, ParticipantTask, Portal, Study } from '../api/api'
 import TaskLink, { getTaskPath, isTaskAccessible, isTaskActive } from './TaskLink'
 import { Link, NavLink } from 'react-router-dom'
 import { DocumentTitle } from 'util/DocumentTitle'
 import { userHasJoinedPortalStudy } from 'util/enrolleeUtils'
 
-import { HubMessageAlert, useHubUpdate } from './hubUpdates'
+import { HubMessageAlert, HubUpdateMessage, useHubUpdate } from './hubUpdates'
 import { filterUnjoinableStudies } from '../Navbar'
+import { ParticipantDashboardAlert, alertDefaults } from '@juniper/ui-core'
 
 
 /** renders the logged-in hub page */
 export default function HubPage() {
-  const { portal } = usePortalEnv()
+  const { portal, portalEnv } = usePortalEnv()
   const { enrollees } = useUser()
+  const [noActivitiesAlert, setNoActivitiesAlert] = useState<ParticipantDashboardAlert>()
+
+  useEffect(() => {
+    loadDashboardAlerts()
+  }, [])
+
+  const loadDashboardAlerts = async () => {
+    try {
+      const alerts = await Api.getPortalEnvDashboardAlerts(portal.shortcode, portalEnv.environmentName)
+      setNoActivitiesAlert({
+        ...alertDefaults['NO_ACTIVITIES_REMAIN'],
+        ...alerts.find(msg => msg.trigger === 'NO_ACTIVITIES_REMAIN')
+      })
+    } catch (error) {
+      console.error(error)
```

Comment:
BLOCKER: we should call either `defaultApiErrorHandle` or `logError` here.  (and JN-750 should handle putting those into a nicer pattern so it's easier to remember that)